bluesky/callbacks/best_effort.py

```diff
@@ -441,6 +441,53 @@ def clear(self):
         self._buffer = StringIO()
         self._baseline_toggle = True
 
+    def plot_prune_fifo(self, num_lines, y_signal, x_signal):
+        """
+        Find the plot with axes x_signal and y_signal.  Replot with only the last *num_lines* lines.
+
+        .. note:: This is not a bluesky plan.  Call it as a normal Python function.
+
+        Example to remove all scans but the last:
+        >>> bec.plot_prune_fifo(1, noisy, m1)
+
+        Parameters
+        ----------
+        num_lines: int
+            number of lines (plotted scans) to keep, must be >= 0
+        y_signal: object
+            instance of ophyd.Signal (or subclass), 
+            dependent (y) axis
+        x_signal: object
+            instance of ophyd.Signal (or subclass), 
+            independent (x) axis
+
+        """
+        assert num_lines >= 0, "num_lines must be 0 or greater"
+        for liveplot in self._live_plots.values():
+            lp = liveplot.get(y_signal.name)
+            if lp is None or lp.x != x_signal.name or lp.y != y_signal.name:
+                continue
+
+            # pick out only the lines that contain plot data
+            # skipping the lines that show peak centers
+            lines = [
+                tr
+                for tr in lp.ax.lines
+                if len(tr._x) != 2 
+                    or len(tr._y) != 2 
+                    or (len(tr._x) == 2 
+                        and tr._x[0] != tr._x[1])
+            ]
+            if len(lines) > n:
+                print(f"limiting LivePlot({y_signal.name}) to {num_lines} lines")
```

Comment:
```suggestion
                print(f"Limiting LivePlot({y_signal.name}) to {num_lines} lines")
```

Reply:
Now it's a complete sentence but it needs to end with a period.  I'll add that.

Reply:
Thanks!